ttnn/cpp/ttnn/tensor/tensor_utils.cpp

```diff
@@ -243,11 +243,11 @@ Helper function to aid in converting grouped weight tensor to ungrouped weight t
 template <typename T>
 static Tensor conv_group_weight_zero_pad_helper(
     Tensor& conv_weight_tensor,
-    tt::tt_metal::LegacyShape& original_weight_shape,
-    tt::tt_metal::LegacyShape& output_weight_shape,
+    ttnn::SimpleShape& original_weight_shape,
+    ttnn::SimpleShape& output_weight_shape,
```

Comment:
we probably must make them `const&` (not a matter of this PR, but while I see it)

Reply:
i can fix. It's trivial